scripts/seed_data.py

```
@@ -0,0 +1,95 @@
+import json
+import os
+from django.utils import timezone
+
+from accounts.models import User, Profile
+from classrooms.models import Classroom, ClassroomTeacher, Enrollment
+
+
+def run():
```

Ruff (0.8.2)

Missing return type annotation for public function `run`

Add return type annotation: `None`

(ANN201)